### PR TITLE
Draft release notes for 0.5.0

### PR DESCRIPTION
**Pull request by jni**

### CODE REVIEW

**Review by Czaki — COMMENTED** [review]
Please update `toc.yml` 

https://github.com/napari/docs/blob/c1bf91b3378b80d3e6bfa75d5684baba1328cffd/docs/_toc.yml#L184

**Comment by Czaki** [comment]
We should add some check to validate if release is added to toc.yml 
@melissawm is it possible to crash build in such a situation?

File: docs/release/release_0_5_0.md [hunk 1]
```diff
@@ -0,0 +1,911 @@
...
+## Build Tools
+
+- ci(dependabot): bump WebFreak001/deploy-nightly from 1.1.0 to 1.2.0 ([napari/napari#5159](https://github.com/napari/napari/pull/5159))
+- ci(dependabot): bump convictional/trigger-workflow-and-wait from 1.6.1 to 1.6.4 ([napari/napari#5293](https://github.com/napari/napari/pull/5293))
+- ci(dependabot): bump bruceadams/get-release from 1.2.3 to 1.3.2 ([napari/napari#5294](https://github.com/napari/napari/pull/5294))
+- ci(dependabot): bump convictional/trigger-workflow-and-wait from 1.6.4 to 1.6.5 ([napari/napari#5384](https://github.com/napari/napari/pull/5384))
+- ci(dependabot): bump WebFreak001/deploy-nightly from 1.2.0 to 2.0.0 ([napari/napari#5385](https://github.com/napari/napari/pull/5385))
+- ci(dependabot): bump docker/build-push-action from 3 to 4 ([napari/napari#5523](https://github.com/napari/napari/pull/5523))
+- ci(dependabot): bump toshimaru/auto-author-assign from 1.6.1 to 1.6.2 ([napari/napari#5524](https://github.com/napari/napari/pull/5524))
+- fix various keybinding/shortcut bugs ([napari/napari#5604](https://github.com/napari/napari/pull/5604))
```
Comment (brisvag) [inline docs/release/release_0_5_0.md]: weird one here

**Review by brisvag — COMMENTED** [review]
I would say a highlight is the shitload of documentation work :P

There's a lot of "others"... is that just missing labels, or really uncaregorizable?

**Comment by jni** [comment]
> There's a lot of "others"... is that just missing labels, or really uncaregorizable?

there are no missing labels (thanks @Czaki!), this is just where maintenance goes. We have discussed the issue of our PR ratio in the core-devs Zulip... 😅

**Comment by jni** [comment]
btw I don't think we have been super diligent with the highlights label, so feel free to suggest other things that you want to tell the community about!

**Comment by melissawm** [comment]
> We should add some check to validate if release is added to toc.yml @melissawm is it possible to crash build in such a situation?

It should be, but unfortunately I haven't been able to solve all other sphinx errors so this error would just die among all the other ~60 we currently have 😭  We are tracking this here: https://github.com/napari/docs/issues/227 and here: https://github.com/napari/docs/pull/409

File: docs/release/release_0_5_0.md [hunk 2]
```diff
@@ -0,0 +1,911 @@
...
+- Add viewer keybinds to select & show only next/prev layer and to toggle visibility of unselected ([napari/napari#6590](https://github.com/napari/napari/pull/6590))
+- Show `About napari` action on macOS application menu ([napari/napari#6666](https://github.com/napari/napari/pull/6666))
+- Add points/shapes highlight color setting to Appereance settings and preference page ([napari/napari#6689](https://github.com/napari/napari/pull/6689))
+- Include features index as key in FormatStringEncoding ([napari/napari#6703](https://github.com/napari/napari/pull/6703))
+- Reset all dims to be rollable ([napari/napari#6797](https://github.com/napari/napari/pull/6797))
+- implement copying spatial information via clipboard ([napari/napari#6864](https://github.com/napari/napari/pull/6864))
+- Add `axis_labels` and `units` to layers and transforms  ([napari/napari#6979](https://github.com/napari/napari/pull/6979))
+- Alpha implementation of NAP-6 ([napari/napari#7011](https://github.com/napari/napari/pull/7011))
+- Screenshot without margins ([napari/napari#6730](https://github.com/napari/napari/pull/6730))
```
Comment (psobolewskiPhD) [inline docs/release/release_0_5_0.md]: Should we rename this? it's not screenshot anymore is it? different API only thing?

File: docs/release/release_0_5_0.md [hunk 3]
```diff
@@ -0,0 +1,911 @@
...
+- Use Dims margin in layer slicing code ([napari/napari#5697](https://github.com/napari/napari/pull/5697))
+- Changing brush size with mouse movement while holding specified modifiers ([napari/napari#5753](https://github.com/napari/napari/pull/5753))
+- Feature: polygon drawing in the Labels layer ([napari/napari#5806](https://github.com/napari/napari/pull/5806))
+- Replace old async loading with new approach ([napari/napari#5816](https://github.com/napari/napari/pull/5816))
+- Enable pinch-to-zoom with vispy 0.13+ ([napari/napari#5859](https://github.com/napari/napari/pull/5859))
+- Keep shading light for Surface fixed relative to camera ([napari/napari#5893](https://github.com/napari/napari/pull/5893))
+- Use Request/Response pattern for Vectors layer ([napari/napari#5918](https://github.com/napari/napari/pull/5918))
+- Lock dimensions / axes while rolling ([napari/napari#5986](https://github.com/napari/napari/pull/5986))
+- Added support for features in surface layers ([napari/napari#6515](https://github.com/napari/napari/pull/6515))
+- Add creating image from clipboard ([napari/napari#6532](https://github.com/napari/napari/pull/6532))
```
Comment (psobolewskiPhD) [inline docs/release/release_0_5_0.md]: To me, this one is 🔥 maybe worth highlight? if doing anything with remote data, like in S3 buckets, it's sick to be able to copy and open directly in napari -- assuming zarr/plugins anyways.

File: docs/release/release_0_5_0.md [hunk 4]
```diff
@@ -0,0 +1,911 @@
...
+- Add viewer keybinds to select & show only next/prev layer and to toggle visibility of unselected ([napari/napari#6590](https://github.com/napari/napari/pull/6590))
+- Show `About napari` action on macOS application menu ([napari/napari#6666](https://github.com/napari/napari/pull/6666))
+- Add points/shapes highlight color setting to Appereance settings and preference page ([napari/napari#6689](https://github.com/napari/napari/pull/6689))
+- Include features index as key in FormatStringEncoding ([napari/napari#6703](https://github.com/napari/napari/pull/6703))
+- Reset all dims to be rollable ([napari/napari#6797](https://github.com/napari/napari/pull/6797))
+- implement copying spatial information via clipboard ([napari/napari#6864](https://github.com/napari/napari/pull/6864))
```
Comment (psobolewskiPhD) [inline docs/release/release_0_5_0.md]: This is also pretty sweet. Not sure how long we want the highlight section, but worth pointing to IMO.

File: docs/release/release_0_5_0.md (hunk 1 above)
Comment (psobolewskiPhD) [reply docs/release/release_0_5_0.md]: The labels:
image

I guess dependencies puts it here?
Maybe tag it with `bugfix`? that seems what it does based on the first post.

File: docs/release/release_0_5_0.md [hunk 5]
```diff
@@ -0,0 +1,911 @@
...
+- ci(dependabot): bump toshimaru/auto-author-assign from 1.6.1 to 1.6.2 ([napari/napari#5524](https://github.com/napari/napari/pull/5524))
+- fix various keybinding/shortcut bugs ([napari/napari#5604](https://github.com/napari/napari/pull/5604))
+- ci(dependabot): bump peter-evans/create-pull-request from 4 to 5 ([napari/napari#5792](https://github.com/napari/napari/pull/5792))
+- ci(dependabot): bump actions/setup-python from 2 to 4 ([napari/napari#6201](https://github.com/napari/napari/pull/6201))
+- ci(dependabot): bump docker/login-action from 2.1.0 to 3.0.0 ([napari/napari#6263](https://github.com/napari/napari/pull/6263))
+- ci(dependabot): bump actions/checkout from 2 to 4 ([napari/napari#6264](https://github.com/napari/napari/pull/6264))
+- ci(dependabot): bump docker/build-push-action from 4 to 5 ([napari/napari#6280](https://github.com/napari/napari/pull/6280))
+- ci(dependabot): bump docker/metadata-action from 4 to 5 ([napari/napari#6282](https://github.com/napari/napari/pull/6282))
+
+## Documentation
```
Comment (psobolewskiPhD) [inline docs/release/release_0_5_0.md:237]: Is it worth breaking out the `maintenance` label docs PRs to a different section?
maybe have a section:
`Copy improvements`
&
`Maintenance`
I can go and do some tagging if needed.

File: docs/release/release_0_5_0.md (hunk 1 above)
Comment (brisvag) [reply docs/release/release_0_5_0.md]: yup, sounds good!

File: docs/release/release_0_5_0.md (hunk 2 above)
Comment (brisvag) [reply docs/release/release_0_5_0.md]: "Export figure" or something like that.

**Comment by Czaki** [comment]
If @jni used `additional_notes/0.5.0` directory in generating notes, the regenerate notes after update labels should be cheap. Just execute the command and wait.

**Comment by jni** [comment]
@Czaki @brisvag could either of you approve this? I've made the suggestions made so far — I'd like to have this up on the site when the alpha is released.

**Comment by jni** [comment]
Well, or @psobolewskiPhD — just realised it's quite late 😂

**Review by Czaki — APPROVED** [review]

**Pull request merged**